pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -490,12 +490,19 @@ func (k *Kubernetes) ConfigEnvs(name string, service kobject.ServiceConfig) []ap
 func (k *Kubernetes) CreateKubernetesObjects(name string, service kobject.ServiceConfig, opt kobject.ConvertOptions) []runtime.Object {
 	var objects []runtime.Object
 	var replica int
+
 	if opt.IsReplicaSetFlag || service.Replicas == 0 {
 		replica = opt.Replicas
 	} else {
 		replica = service.Replicas
 	}
 
+	// Check to see if Docker Compose v3 Deploy.Mode has been set to "global"
+	if service.DeployMode == "global" {
+		log.Infof("Global mode has been set. Containers will only be replicated once throughout the cluster")
```

Comment:
> Would it be better to output a warning saying that Global is not support, but replicas will be set to 1?

Yes , I think this would be the best.

We can remove it once we start doing the DaemonSet thing. ( automatically using DaemonSets for services with global mode.)